website/content/docs/connect/cluster-peering/k8s.mdx

```diff
@@ -0,0 +1,211 @@
+---
+layout: docs
+page_title: Cluster Peering on Kubernetes
+description: >-
+  This page describes how to create peering connections, deploy services, export cluster services, and end peering connections for Consul cluster peering using Kubernetes (K8s).
+---
+
+# Cluster Peering on Kubernetes
+
+~> This page covers features that are currently in _technical preview_. Features and functionality are subject to change. You should never use the technical preview release in secure environments or production scenarios. Features in technical preview may face performance and scaling issues, with limited support options available.
+
+To establish a cluster peering connection on Kubernetes, you need to enable the feature in the Helm chart and create custom resource definitions for each side of the peering.
+
+The following Custom Resource Definitions (CRDs) are used to create and manage a peering connection:
+
+- `PeeringAcceptor`: Generates a peering token and accepts an incoming peering connection.
+- `PeeringDialer`: Uses a peering token to make an outbound peering connection with the cluster that generated the token.
+
+## Prerequisites
+
+To create and use cluster peering connections with Kubernetes, you need at least two Kubernetes clusters running in a flat network with Consul on Kubernetes v. 0.45 or later.
+
+### Helm chart configuration
+To establish cluster peering through Kubernetes, deploy clusters with the following Helm values.
+
+```yaml
+global:
+  peering:
+    enabled: true
+connectInject:
+  enabled: true
+meshGateway:
+  enabled: true
+  replicas: 1
+```
+
+## Create a peering connection
+
+To peer Kubernetes clusters running Consul, you need to create a peering token and share it with the other cluster.
+
```

Comment:
```suggestion

### Cluster 1
```

Reply:
This is a suggestion to add a subheader per cluster to be extra clear, but feel free to not use if you think it'll clutter the docs